test/test_other.py

```diff
@@ -2886,6 +2886,12 @@ def test_jspi_wildcard(self):
 
     self.do_runf(test_file('other/test_jspi_wildcard.c'), 'done')
 
+  def test_embind_tsgen(self):
+    self.run_process([EMCC, test_file('other/embind_tsgen.cpp'),
+                      '-lembind', '--embind-emit-tsd', 'embind_tsgen.d.ts.out'], stdout=PIPE)
```

Comment:
Why not just call the output file `embind_tsgen.tsd`?   Is there standard file extension for TS types?

Reply:
` .d.ts` is the standard.